AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/Supporting Files/Banners.storyboard

```diff
@@ -16,22 +17,22 @@
                         <viewControllerLayoutGuide type="bottom" id="amy-cz-zWv"/>
                     </layoutGuides>
                     <view key="view" clipsSubviews="YES" contentMode="scaleToFill" id="7X4-aq-Zti">
-                        <rect key="frame" x="0.0" y="0.0" width="414" height="759"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="837"/>
```

Comment:
Is there a reason for these storyboard changes? Looks like we are modifying the other banner types as well, not just the newly added SwiftUI banners. Same comment for Leaders/MRECs storyboards.

Reply:
The code snippet you highlighted is the Storyboard change for the table view of the different implementations of the AppLovin banners as seen in the screenshot below. 

![Screenshot 2023-08-24 at 4 08 56 PM](https://github.com/AppLovin/AppLovin-MAX-SDK-iOS/assets/75236585/0b28572d-e9c5-4967-83c4-b506bd0621fe)

The storyboard changes are to add the SwiftUI implementation in the table view menu for each format while also adding the SwiftUI view controller to contain the SwiftUI view. The AppLovin network ad formats have storyboards separated by the ad formats, while the MAX ad formats are all housed in `Main.storyboard`